appalerts/templates/appalerts/alert_list.html

```diff
@@ -23,21 +23,45 @@
     .sodar-app-alert-row div:nth-child(1) {
         max-width: 40px;
     }
+
+    .sodar-app-alert-btn {
+        height: 30px;
+    }
   </style>
 {% endblock css %}
 
 {% block projectroles %}
 
 <div class="row sodar-subtitle-container">
   <h2><i class="iconify" data-icon="mdi:alert-octagram"></i> App Alerts</h2>
-  <a class="btn btn-primary ml-auto sodar-app-alert-btn-dismiss
-           {% if object_list.count == 0 %}disabled{% endif %}"
-     id="sodar-app-alert-btn-dismiss-all"
-     role="button"
-     data-uuid=""
-     data-dismiss-url="{% url 'appalerts:ajax_dismiss_all' %}">
-    <i class="iconify" data-icon="mdi:close-thick"></i> Dismiss All
-  </a>
+  {% if not dismissed %}
+    <div class="btn-group sodar-list-btn-group ml-auto">
+      <button class="btn btn-primary dropdown-toggle sodar-app-alert-btn"
+              id="sodar-app-alert-btn-dropdown-operations"
+              type="button" data-toggle="dropdown" aria-expanded="false">
+        <i class="iconify" data-icon="mdi:cog"></i>Alert Operations
+      </button>
+      <div class="dropdown-menu dropdown-menu-right">
+          <a class="dropdown-item" id="sodar-app-alert-btn-dismissed"
+             href="{% url 'appalerts:list_dismissed' %}">
+            <i class="iconify" data-icon="mdi:alert-octagram"></i> Dismissed Alerts
```

Comment:
I recommend labeling this "View Dismissed" to convey an action. Also, it might be clear if the icon would be different from default. I recommend `mdi:alert-octagram-outline`. We should also display the alternate icon in the `<h2>` tag if in dismissed mode.